README.rst

```diff
@@ -12,20 +12,14 @@ Building Locally
 Install Prerequisites
 ~~~~~~~~~~~~~~~~~~~~~
 
-Debian 11 (“testing”) / Ubuntu 20.04 or later::
-
-   sudo apt install python3-sphinx python3-sphinx-issues python3-sphinx-rtd-theme
-
-Debian 10 (”Buster”) and earlier::
+::
 
+   # For Debian based distros
    sudo apt install pip3
-   pip3 install -r requirements.txt
-   # Add ~/.local/bin to your $PATH, e.g. by adding this to ~/.bashrc:
-   PATH=$HOME/.local/bin:$PATH
-
-Other distros::
+   # For RPM-based distros
```

Comment:
There are ready to use rpm packages in Fedora - `python3-sphinx`, `python3-sphinx-issues`, `python3-sphinx_rtd_theme`.

Reply:
There are some modules that don't exist in distro packages, that's why I think it's easier to just install requirements via `pip install -r requirements.txt`

Reply:
Specifically, it's module from https://github.com/openzfs/openzfs-docs/pull/14

Reply:
There is also a package for that module in Fedora `python3-sphinx-notfound-page.noarch`. :)

Reply:
@mskarbek ah, Fedora :) I looked at base centos repo.

Main point - it depends :) So better to give at least one 99% working solution.

Reply:
I may bring some other modules soon, let's get back to this queston in a month to have a minimally settled dependencies as a start, otherwise it may be a pointless time. 

Maybe it would be easier to work in venv, so we may just add one `make me happy` command to prepare venv, deps and generate docs there.

Reply:
@mskarbek
> There are ready to use rpm packages in Fedora - `python3-sphinx`, `python3-sphinx-issues`, `python3-sphinx_rtd_theme`.

I'm not finding these on: https://src.fedoraproject.org From that site, it looks like maybe they don't have the "3":
```
  sudo yum install python-sphinx python-sphinx-issues \
      python-sphinx_rtd_theme
```